R/init.R

```diff
@@ -4,44 +4,43 @@
 # error messages.
 
 
-#' Create the Server and UI Function For the Shiny App
+#' Create the server and UI function for the `shiny` app
 #'
 #' @description `r lifecycle::badge("stable")`
+#'
 #' End-users: This is the most important function for you to start a
-#' teal app that is composed out of teal modules.
+#' `teal` app that is composed out of `teal` modules.
 #'
 #' @details
 #' When initializing the `teal` app, if `datanames` are not set for the `teal_data` object,
 #' defaults from the `teal_data` environment will be used.
 #'
-#' @param data (`teal_data`, `teal_data_module`, `named list`)\cr
-#' `teal_data` object as returned by [teal.data::teal_data()] or
-#' `teal_data_module` or simply a list of a named list of objects
-#' (`data.frame` or `MultiAssayExperiment`).
-#' @param modules (`list`, `teal_modules` or `teal_module`)\cr
+#' @param data (`teal_data` or `teal_data_module`)
+#' `teal_data` object as returned by [`teal.data::teal_data()`] or `teal_data_module`.
+#' @param modules (`list` or `teal_modules` or `teal_module`)
 #'   nested list of `teal_modules` or `teal_module` objects or a single
 #'   `teal_modules` or `teal_module` object. These are the specific output modules which
-#'   will be displayed in the teal application. See [modules()] and [module()] for
+#'   will be displayed in the `teal` application. See [`modules()`] and [`module()`] for
 #'   more details.
-#' @param title (`shiny.tag` or `character(1)`)\cr
+#' @param title (`shiny.tag` or `character(1)`)
 #'   The browser window title. Defaults to a title "teal app" with the icon of NEST.
 #'   Can be created using the `build_app_title()` or
 #'   by passing a valid `shiny.tag` which is a head tag with title and link tag.
-#' @param filter (`teal_slices`)\cr
-#'   Specification of initial filter. Filters can be specified using [teal::teal_slices()].
+#' @param filter (`teal_slices`)
+#'   Specification of initial filter. Filters can be specified using [`teal::teal_slices()`].
```

Comment:
redundant if to self

```suggestion
#'   Specification of initial filter. Filters can be specified using [`teal_slices()`].
```

Reply:
```suggestion
#'   Initial filter specification.
```

Reply:
done [91a698e](https://github.com/insightsengineering/teal/pull/1021/commits/91a698ef71e608a630a016e224f9dff44eb0b320), Also moved additional information in details.